plan/aggregation_push_down.go

```diff
@@ -295,6 +327,17 @@ func (a *aggPushDownSolver) aggPushDown(p LogicalPlan) {
 				rChild.SetParents(join)
 				join.SetSchema(append(lChild.GetSchema().Clone(), rChild.GetSchema().Clone()...))
 			}
+		} else if union, ok1 := child.(*Union); ok1 {
+			pushedAgg := a.makeNewAgg(agg.AggFuncs, agg.groupByCols)
+			oldSchema := union.schema
+			union.SetSchema(pushedAgg.schema)
```

Comment:
Maybe update union schema after push aggregate to children is better?
So we don't to define `oldSchema`.